rest/handler.go

```diff
@@ -224,7 +224,7 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []string, respo
 		if dbContext != nil {
 			managementEndpoints, httpClient, err = dbContext.ObtainManagementEndpointsAndHTTPClient()
 		} else {
-			managementEndpoints, httpClient, err = h.server.ObtainManagementEndpointsAndHTTPClient()
+			managementEndpoints, httpClient, err = h.server.ObtainManagementEndpointsAndHTTPClient(dbContext.BucketSpec.CACertUnsetTlsSkipVerify)
```

Comment:
Since dbContext == nil here, we can't use it to get the value of tlsSkipVerify.  Might need additional testing for this path. 

@JRascagneres Does this imply that we also need a server-level unsupported insecureSkipVerify flag, with the option to override at the db level?

Reply:
@adamcfraser Eventually this option is going to be at the server level. Especially this is backed up by Bens work. There is only one server that the persistent config setup can connect to and this server address cannot be overridden at the db level.